src/states_screens/race_gui_overworld.cpp

```diff
@@ -443,6 +443,17 @@ void RaceGUIOverworld::drawGlobalMiniMap()
         const ChallengeData* challenge = unlock_manager->getChallengeData(challenges[n].m_challenge_id);
         const unsigned int val = challenge->getNumTrophies();
         bool unlocked = (PlayerManager::getCurrentPlayer()->getPoints() >= val);
+        if (challenges[n].m_challenge_id == "fortmagma") {
+            // For each track, check whether the easy difficulty has been completed ; fortmagma will not affect our decision (`n == m`) ; tutorial is ignored because it has no completion level
```

Comment:
Should say "check whether any difficulty"...

Reply:
Correct. I forgot to change the comment.